Source/WebCore/bindings/js/JSDOMGlobalObject.cpp

```diff
@@ -498,32 +519,34 @@ JSC::JSPromise* JSDOMGlobalObject::instantiateStreaming(JSC::JSGlobalObject* glo
 }
 #endif
 
-static ScriptModuleLoader* scriptModuleLoader(JSDOMGlobalObject* globalObject)
+ScriptModuleLoader* JSDOMGlobalObject::scriptModuleLoader() const
```

Comment:
does making this a method of JSDOMGlobalObject actually change anything?

Reply:
it exposes it beyond this file--we're using this from `JSShadowRealmGlobalScopeBase::finishCreation` to build the new module loader for the realm

Reply:
That's true, but it's a lot of refactoring to enable that, so I think there are alternatives we could try which make the diff smaller. I'm not going to push back too hard on it though 👍

Reply:
OK fair, i'll look again--this seemed to me at the time to be the easiest way out but maybe I missed something